docs/en/concepts-and-designs/oal.md

```diff
@@ -34,7 +34,7 @@ Read [Scope Definitions](scope-definitions.md), you can find all existing Scopes
 Use filter to build the conditions for the value of fields, by using field name and expression. 
 
 The expressions support to link by `and`, `or` and `(...)`. 
-The OPs support `==`, `!=`, `>`, `<`, `>=`, `<=`, `in [...]` ,`like %...`, `like ...%` and `like %...%`, with type detection based of field type. Trigger compile
+The OPs support `==`, `!=`, `>`, `<`, `>=`, `<=`, `in [...]` ,`like %...`, `like ...%` , `like %...%` , `inclueds` and `excludes`, with type detection based of field type. Trigger compile
```

Comment:
Hi, there is a word chosen game here. :) Take a look about this, https://ell.stackexchange.com/questions/47706/using-contain-vs-include-vs-consist-of-appropriately#:~:text=The%20distinction%20between%20'includes'%20and,container%2C%20and%20contains%20the%20hats.&text=Inside%20the%20box%20is%20a,and%20consists%20of)%20the%20hats.

And check the JDK API. For a list, we should use `contain`. And don't use `contains`, if you look at the OAL scripts, `tags contains` is wrong in English grammar.

Reply:
Do you face trouble if use `not contain` as an operator?

Reply:
yes,  some segments will not have tags sometimes. So I provided excludes solving this case.

Reply:
I actually first using Contains rather than includes. but not contains have 2 words. so I choose excludes and includes as last.

Reply:
`not contain` seems very clear, in SQL, you have `not in`, right?

Reply:
ok, I will change the words to contain and not contain now.